src/CardView/CardView.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  View,
+  ImageBackground,
+  Image,
+  Text,
+  Platform,
+} from 'react-native'
+
+import defaultIcons from '../Icons'
+import FlipCard from 'react-native-flip-card'
+import styles from './styles'
+
+const BASE_SIZE = { width: 300, height: 190 }
```

Comment:
@yuritoledo I didn't manage to see the code running. Don't you think it's a good idea to set the width and height of the screen? We can do that using the Dimensions from react native.

https://facebook.github.io/react-native/docs/dimensions

Reply:
I agree with you, but in this PR, I just refactor it to functional, I didn't focus on business logic.

We can add some `// TODO` into pieces of code like this to know where we can improve.